pkg/core/block/header.go

```diff
@@ -13,6 +13,9 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/util"
 )
 
+// Version is a default Neo block version.
+const Version uint32 = 0
```

Comment:
We gonna have version 1 soon, so what `block.Version` would mean?

Reply:
I think we inevitably need constants for version, going with literals seems not good, especially with the upcoming version 1. How about adding `Version0` and `Version1` constants? Or we may even create a special type for version. What would you prefer?

Reply:
I think they'll be tied to hardforks. We don't yet have the next name, but current one can be `VersionInitial`. Then it'll be like `VersionDsmth`.

Reply:
Fixed.